internal/app/machined/pkg/controllers/cri/image_cache_config.go

```diff
@@ -252,7 +252,12 @@ func (ctrl *ImageCacheConfigController) createVolumeConfigISO(ctx context.Contex
 		volumeCfg.TypedSpec().Locator = block.LocatorSpec{
 			Match: *boolExpr,
 		}
-		volumeCfg.TypedSpec().Mount.TargetPath = constants.ImageCacheISOMountPoint
+		volumeCfg.TypedSpec().Mount = block.MountSpec{
+			TargetPath: constants.ImageCacheISOMountPoint,
+			FileMode:   0o700,
+			UID:        0,
+			GID:        0,
+		}
```

Comment:
I guess these also needs proper selinux label :thinking:

Reply:
@dsseng is there any proper SELinux label for the image cache data, or keep it undefined?

Reply:
We consider it to be world-readable? Let's create a new context for this and add to common_f typeattribute

Reply:
i guess only machined needs access to this